owl/list.scm

```diff
@@ -222,15 +206,17 @@
                ((pred (car lst)) (loop (cdr lst) (cons (car lst) taken)))
                (else (values (reverse taken) lst)))))
 
-      (define (keep pred lst)
```

Comment:
Filter is an amazingly bad choice for a standard function name, but it is standard, so it's ok to go with it. 

Keep and remove clearly declare what they want to end up with. Filtering is just a process of separating something from something. It's not clear whether you want the filter to collect the stuff you want or remote the stuff you don't want.

Reply:
Are you interested in taking part in developentment more officially? I'm thinking of transferring owl to an organization user, so that it would be easier to add members with different roles.

Reply:
Yes, that sounds great.